samples/tflm_ethosu/linker.ld

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Arm Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(ddr4), okay)
+GROUP_START(DDR4)
+
+	SECTION_DATA_PROLOGUE(_DDR4_SECTION_NAME,,SUBALIGN(16))
+	{
+        . = ALIGN(16);
+        *(tflm_model tflm_arena tflm_input tflm_output)
```

Comment:
please fix alignment here, this is mixing tabs/spaces

Reply:
Will fix.